tools/sink-analyser/retrieve_project_link.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2023 Fuzz Introspector Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#!/usr/bin/python3
+
+import requests
+import lxml.html
+import json
+
+data = requests.get('https://oss-fuzz-introspector.storage.googleapis.com/index.html').text
+html = lxml.html.fromstring(data)
+
+link_map = dict()
+for li in html.getchildren()[1].getchildren()[2]:
```

Comment:
The HTML changed recently and looks different now: https://oss-fuzz-introspector.storage.googleapis.com/index.html -- I assume the HTML  change may have broken the script?

Am not sure what the smartest thing is here in order to make it more general. However, for now, could you update the script so it works with the latest?

Reply:
changed